config/initializers/job_configurations.rb

```diff
@@ -116,6 +116,18 @@
         cron: IdentityConfig.store.get_usps_proofing_results_job_cron,
         args: -> { [Time.zone.now] },
       },
+      # Queue usps proofing job to GoodJob for ready enrollments
+      get_usps_ready_proofing_results_job: {
+        class: 'GetUspsReadyProofingResultsJob',
+        cron: IdentityConfig.store.get_usps_ready_proofing_results_job_cron,
+        args: -> { [Time.zone.now] },
+      },
+      # Queue usps proofing job to GoodJob for waiting enrollments
+      get_usps_ready_proofing_results_job: {
```

Comment:
should this be `GetUspsWaitingProofingResultsJob` ?

Reply:
Yep! good catch. Thanks!